src/WebApp/appsettings.json

```diff
@@ -0,0 +1,25 @@
+{
+    "Logging": {
+        "Default": "Debug",
+        "Microsoft": "Warning",
+        "Microsoft.Hosting.Lifetime": "Information"
+    },
+    "IdentityServer": {
+        "Clients": [
+            {
+                "ClientId": "codidact.com",
+                "ClientSecrets": [
+                    {
+                        "Value": "LCa0a2j/xo/5m0U8HTBBNBNCLXBkg7+g+YpeiGJm564="
+                    }
+                ],
+                "AllowedGrantTypes": [
+                    "client_credentials"
```

Comment:
Why did you switch to client credentials over PKCE authorization code?
PKCE/Code is the recommended way.

I think you are exposing your client secret on every login request too?
Client Credentials is not supposed to be used between websites verification like this

Reply:
You are right. This should be `authorization_code`.

f930142c